src/ctap/mod.rs

```diff
@@ -1342,8 +1355,19 @@ mod test {
     use cbor::{cbor_array, cbor_array_vec, cbor_map};
     use crypto::rng256::ThreadRng256;
 
-    const CLOCK_FREQUENCY_HZ: usize = 32768;
-    const DUMMY_CLOCK_VALUE: ClockValue = ClockValue::new(0, CLOCK_FREQUENCY_HZ);
+    const CLOCK_FREQUENCY_HZ: u32 = 32768;
+    #[derive(Default, Clone, Copy)]
+    pub struct TestClock();
```

Comment:
We could move all `Clock` implementations to a separate file, and this one conditionally compiles only for `#[cfg(test)]`. Then all files that need the `TestClock` can import it, and main is less cluttered.

Reply:
Done.